mgmt/src/mgmt.c

```diff
@@ -74,7 +74,7 @@ mgmt_streamer_free_buf(struct mgmt_streamer *streamer, void *buf)
 void
 mgmt_unregister_group(struct mgmt_group *group)
 {
-    struct mgmt_group *curr = mgmt_group_list, *prev;
+    struct mgmt_group *curr = mgmt_group_list, *prev = NULL;
```

Comment:
This might fix the warning but it still could break when doing the `prev->mg_next = curr->mg_next;` below. I submitted a review for the original PR and would prefer a fix like suggested.